install/0000_00_cluster-version-operator_01_admingate_configmap.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: admin-gates
+  namespace: openshift-cluster-version
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+data:
+  ack-4.8-kube-122-api-removals-in-4.9: |
+    Detailed description of what admin needs to do...Read documentation here https://
```

Comment:
probably needs to stay WIP until we have some content to put here?

Reply:
Actually I need to pull the data out since we have no active gate until back port to 4.8.

Reply:
Removed the data altogether since there are no active gates in 4.9.